blaze/server/server.py

```diff
@@ -1,6 +1,10 @@
 from __future__ import absolute_import, division, print_function
 
 import socket
+try:
+    import cPickle as pickle
+except ImportError:
+    import pickle
```

Comment:
sorry, I misunderstood and thought you meant some compatibility requirements file.